### ISSUE TITLE
Comparisons between pump.io and X

### ISSUE THREAD

**Issue by Serkan-devel** [pump-io/pump.io#1487]
It would be great to have a wiki entry, comparing pump.io with other open source social network like Mastodon, Diaspora or Minds.

**Comment by vxcamiloxv** [comment]
A comparison should be hard, varies too much according with preference, audience, interface, I thinks a comparison of which one is better is not cool, but should exist a feature list


Easy to install
Low consume
Variety of mobile/desktop clients
etc.

**Comment by Serkan-devel** [comment]
No, I mean comparisons, like `programming language`,  `system requirements` or other features.

**Comment by vxcamiloxv** [comment]
Ha ok, it could be useful.

**Comment by strugee** [comment]
Would love to see either a wiki page for this or even better a page in the documentation ;)

https://gitlab.com/pump.io/pump-docs
